sympy/integrals/integrals.py

```diff
@@ -6,7 +6,7 @@
 from sympy.core.compatibility import is_sequence, range
 from sympy.core.containers import Tuple
 from sympy.core.expr import Expr
-from sympy.core.function import diff
+from sympy.core.function import diff, count_ops
```

Comment:
I would follow the example of `Sum._eval_simplify` and just leave measure=None in the arg list...the calling program will be passing the measure function.